source/developer/localization-process.rst

```diff
@@ -0,0 +1,116 @@
+Localization
+------------------
+
+The goal of the localization process is to consistently produce high quality translations release after release through collaboration between the international community, translators, project managers, language leads and the core team.
+
+Translation Process
+===================
+
+If you're interested in contributing translations to Mattermost, please join the `Mattermost localization channel to discuss <https://pre-release.mattermost.com/core/channels/localization>`_. Below are descriptions of the overall localization process.
```

Comment:
Seems the `_` before the fullstop at the end of the first sentence shouldn't be there?